crates/sui-kvstore/src/bigtable/client.rs

```diff
@@ -423,6 +434,47 @@ impl BigTableClient {
         &mut self,
         table_name: &str,
         keys: Vec<Vec<u8>>,
+    ) -> Result<Vec<Vec<(Bytes, Bytes)>>> {
+        let elapsed = Instant::now().elapsed();
+        let num_keys_requested = keys.len();
+        let result = self.multi_get_internal(table_name, keys).await;
+        let labels = [&self.client_name, table_name];
+        match &self.metrics {
```

Comment:
I think having metrics in `multi_get` labeled by table is very useful. But I wonder if it would be useful to all group by method because we have multiple methods that may hit the same table, like `get_latest_object` and `get_objects` and it would be useful to know if their metrics are somehow different.

Reply:
This PR only covers `multi_get`, and there is only one method per column family. 
`reversed_scan` can be added as followup. 
Just curious, why the indexer needs to rely on the KV store for get_latest_object call?

Reply:
You are right. Indexer does not need `get_latest_object` calls. We should good with `multi_get` metrics.